src/backend/parser/cypher_clause.c

```diff
@@ -5093,7 +5131,7 @@ transform_cypher_create_path(cypher_parsestate *cpstate, List **target_list,
         else
         {
             ereport(ERROR,
-                   (errmsg_internal("unrecognized node in create pattern")));
+                    (errmsg_internal("unrecognized node in create pattern")));
```

Comment:
[formatting] extra whitespace ?

Reply:
Actually, that is the correct indentation, extra space. Parameters for same function line up under each other when 80 characters is exceeded.